stories/core_components/BarChart.tsx

```diff
@@ -323,4 +324,42 @@ storiesOf('Core components/BarChart', module)
                 />
             </div>
         )
+    ))
+    .add('dual axes with two bottom measures, one top measure, one attribute', () => (
+        screenshotWrap(
+            <div style={wrapperStyle}>
+                <BarChart
+                    projectId="storybook"
+                    measures={[MEASURE_1, MEASURE_2, MEASURE_3]}
+                    config={{
+                        secondary_xaxis: {
+                            measures: [MEASURE_3.measure.localIdentifier]
```

Comment:
try to put here M2 identifier and observe result

Reply:
could you point me the issue you see?
you mean column M2 should be displayed after M3?